content/en/docs/contribute/participating.md

```diff
@@ -32,65 +32,61 @@ aspects of Kubernetes -- the Kubernetes website and documentation.
 
 {{% capture body %}}
 
-## Roles and responsibilities
+## Roles overview
 
-When a pull request is merged to the branch used to publish content (currently
-`master`), that content is published and available to the world. To ensure that
-the quality of our published content is high, we limit merging pull requests to
-SIG Docs approvers. Here's how it works.
+- **Anyone** can contribute to Kubernetes documentation. To contribute, you must [sign the CLA](/docs/contribute/start#sign-the-cla) and have a GitHub account.
+- **Members** of the Kubernetes organization are contributors who have spent time and effort on some area of the Kubernetes project, usually in the form of opening pull requests. See [Community membership](https://github.com/kubernetes/community/blob/master/community-membership.md) for membership criteria.
+- A SIG Docs **Reviewer** is a member of the Kubernetes organization who has
+  expressed interest in reviewing documentation pull requests, and has been
+  added to the appropriate GitHub group and `OWNERS` files in the GitHub
+  repository by a SIG Docs Approver.
+- A SIG Docs **Approver** is a member in good standing who has shown a continued
+  commitment to the project. An approver can merge pull requests
+  and publish content on behalf of the Kubernetes organization.
+  Approvers can also represent SIG Docs in the larger Kubernetes community.
+  Some of the duties of a SIG Docs approver, such as coordinating a release,
+  require a significant time commitment.
 
-- When a pull request has both the `lgtm` and `approve` labels and has no `hold`
-  labels, the pull request merges automatically. 
-- Kubernetes organization members and SIG Docs approvers can add comments to
-  prevent automatic merging of a given pull request (by adding a `/hold` comment
-  or withholding a `/lgtm` comment).
-- Any Kubernetes member can add the `lgtm` label, by adding a `/lgtm` comment.
-- Only an approver who is a member of SIG Docs can cause a pull request to merge
-  by adding an `/approve` comment. Some approvers also perform additional
-  specific roles, such as [PR Wrangler](#pr-wrangler) or
-  [SIG Docs chairperson](#sig-docs-chairperson).
+## Anyone
 
-For more information about expectations and differences between the roles of
-Kubernetes organization member and SIG Docs approvers, see
-[Types of contributor](/docs/contribute#types-of-contributor). The following
-sections cover more details about these roles and how they work within
-SIG Docs.
+Anyone can do the following:
 
-### Anyone
+- File a GitHub issue against any part of Kubernetes, including documentation.
+- Provide non-binding feedback on a pull request
```

Comment:
For bulleted items, use periods consistently or not at all. Either choice is potentially valid; consistency is the key.